pepper-apis/dss-core/src/main/java/org/broadinstitute/ddp/copy/CopyExecutor.java

```diff
@@ -56,13 +60,32 @@ public void execute(Handle handle, long operatorId, long participantId, CopyConf
             CopyLocation target = pair.getTarget();
 
             if (source.getType() == CopyLocationType.ANSWER && target.getType() == CopyLocationType.ANSWER) {
+                QuestionDto sourceQuestion;
+                FormResponse sourceInstance;
                 String sourceStableId = ((CopyAnswerLocation) source).getQuestionStableId();
-                QuestionDto sourceQuestion = questionDtosByStableId.get(sourceStableId);
-                if (sourceQuestion == null) {
-                    continue; // Question might have been removed from activity, so we skip it.
+                String user = ((CopyAnswerLocation) source).getUser();
+                if (UserType.OPERATOR.equalsIgnoreCase(user)) {
+                    Stream<Governance> governances = handle.attach(UserGovernanceDao.class)
+                            .findActiveGovernancesByParticipantAndStudyIds(participantId, config.getStudyId());
+                    Optional<Governance> optionalGovernance = governances.findFirst();
+                    if (optionalGovernance.isEmpty()) {
+                        throw new DDPException(String.format("Governance not found for participant %s", participantId));
+                    }
+                    Governance governance = optionalGovernance.get();
```

Comment:
Given that `governances` is not used later in this scope, the intermediate variables can be removed and the code simplified to something like the suggested change.
```suggestion
                    var governance = handle.attach(UserGovernanceDao.class)
                            .findActiveGovernancesByParticipantAndStudyIds(participantId, config.getStudyId())
                            .findFirst()
                            .orElseThrow(() -> {
                                return new DDPException(String.format("Governance not found for participant %s", participantId));
                            });
```